### PR TITLE
Add link to traces in tree benchmarks

### PR DESCRIPTION
**Pull request by icristescu**

### CODE REVIEW

**Review by Ngoguey42 — APPROVED** [review]

File: bench/irmin-pack/tree.ml [hunk 1]
```diff
@@ -1233,5 +1233,22 @@ let main_term =
...
-  let info = Term.info "Benchmarks for tree operations" in
+  let man =
+    [
+      `S "DESCRIPTION";
+      `P
+        "Benchmarks for tree operations. Requires traces of operations, \
+         download them (`wget trace.repr`) from: ";
+      `P
+        "Trace with $(b,10310) commits \
+         https://s3.fr-par.scw.cloud/data.tarides.com/irmin/data4_10310commits.repr";
```
Comment (samoht) [inline bench/irmin-pack/tree.ml:1244]:
```suggestion
         http://data.tarides.com/irmin/data4_10310commits.repr";
```

File: bench/irmin-pack/tree.ml [hunk 2]
```diff
@@ -1233,5 +1233,22 @@ let main_term =
...
+      `P
+        "Trace with $(b,100066) commits \
+         https://s3.fr-par.scw.cloud/data.tarides.com/irmin/data4_100066commits.repr";
```
Comment (samoht) [inline bench/irmin-pack/tree.ml:1247]:
```suggestion
         http://data.tarides.com/irmin/data4_100066commits.repr";
```

File: bench/irmin-pack/tree.ml [hunk 3]
```diff
@@ -1233,5 +1233,22 @@ let main_term =
...
+      `P
+        "Trace with $(b,654941) commits \
+         https://s3.fr-par.scw.cloud/data.tarides.com/irmin/data4_654941commits.repr";
```
Comment (samoht) [inline bench/irmin-pack/tree.ml:1250]:
```suggestion
         http://data.tarides.com/irmin/data4_654941commits.repr";
```

File: bench/irmin-pack/tree.ml [hunk 4]
```diff
@@ -1233,5 +1233,22 @@ let main_term =
...
+      `P
+        "Trace with $(b,10310) commits \
+         http://data.tarides.com/irmin/data4_10310commits.repr";
+      `P
+        "Trace with $(b,100066) commits \
+         http://data.tarides.com/irmin/data4_100066commits.repr";
+      `P
+        "Trace with $(b,654941) commits \
+         http://data.tarides.com/irmin/data4_654941commits.repr";
+    ]
```
Comment (samoht) [inline bench/irmin-pack/tree.ml:1251]: would be nice to point to the trace format specification too (when this spec exists :p)

Comment (Ngoguey42) [reply bench/irmin-pack/tree.ml:1251]: Soooon

**Pull request merged**